source/styleEditor.js

```diff
@@ -1,3 +1,11 @@
 $(document).ready(function(){
   // your code goes here.
+  $('#style_editor input[type="submit"]').click(function(){
+    event.preventDefault();
+    var selector = $(this).parent().find('input[name="selector"]').val()
```

Comment:
Nifty DOM traversal here, but you can also get these elements with something like `$("input[name='selector']")`.